app/src/main/java/com/practicum/playlistapp/SettingsActivity.kt

```diff
@@ -15,6 +17,32 @@ class SettingsActivity : AppCompatActivity() {
         back.setOnClickListener {
             finish()
         }
+        val shareButton = findViewById<Button>(R.id.share)
+        shareButton.setOnClickListener {
+            val share = Intent(Intent.ACTION_SENDTO)
+            val sms = R.string.LinkAndroid
+            share.data = Uri.parse("smsto:")
+            share.putExtra(Intent.EXTRA_TEXT, "sms")
```

Comment:
Здесь необходимо передать строку по id из переменной `sms`:
```suggestion
            share.putExtra(Intent.EXTRA_TEXT, getString(sms))
```